setup/dkim.sh

```diff
@@ -64,6 +64,32 @@ tools/editconf.py /etc/opendmarc.conf -s \
 	"Syslog=true" \
 	"Socket=inet:8893@[127.0.0.1]"
 
+# SPFIgnoreResults causes the filter to ignore any SPF results in the header
+# of the message. This is useful if you want the filter to perfrom SPF checks
+# itself, or because you don't trust the arriving header. This added header is
+# used by spamassassin to evaluate the mail for spamminess.
+
+tools/editconf.py /etc/opendmarc.conf -s \
+        "SPFIgnoreResults=true"
```

Comment:
The original format of `opendmarc.conf` and `opendkim.conf` use a space instead of `=`. The `opendmarc.conf(5)` and `opendkim.conf(5)` man pages state to use white space, and I actually don't know if the `=` is the same as a space for this purpose.

> Blank  lines  are ignored.  Lines containing a hash ("#") character are
> truncated at the hash character to allow for comments in the file.
> 
> Other content should be the name of  a  parameter,  followed  by  white
> space,  followed  by  the  value  of that parameter, each on a separate
> line.
> 
> For parameters that are Boolean in nature, only the first byte  of  the
> value  is  processed.  For positive values, the following are accepted:
> "T", "t", "Y", "y",  "1".   For  negative  values,  the  following  are
> accepted: "F", "f", "N", "n", "0".

Not sure if this makes any difference, just noting it.

Reply:
Take a look at the -s argument for editconf.py

https://github.com/mail-in-a-box/mailinabox/blob/8b166f30415611c6f3c846065a1b9a67c5d12de2/tools/editconf.py#L13